tests/testdata/test/load_unload.out

```diff
@@ -1,6 +1,16 @@
-Check [WILDCARD]/test/load_unload.ts
-running 1 test from ./test/load_unload.ts
+Check [WILDCARD]/load_unload.ts
+------- pre-test output -------
+load
+----- output end -----
+running 1 test from [WILDCARD]/load_unload.ts
+test ...
+------- output -------
+test
+----- output end -----
 test ... ok ([WILDCARD])
+------- post-test output -------
+unload
+----- output end -----
```

Comment:
This is a bit confusing - we have `output end` two end. Is this actually expected?

Reply:
The `output end` matches with `post-test output`. 

Maybe I can clarify this by making the pairs `------- post-test output -------` and `------- post-test output end -------` instead. WDYT?

Reply:
I think it will be less confusing, so +1 from me